### ISSUE TITLE
Bias Corrected precipitation and temperature

### ISSUE THREAD

**Issue by theobarnhart-USGS** [hytest-org/hytest#176]
Hello, 

Thank you for all of the information lately about the CONUS404 dataset. I've been working with the `conus404-hourly-onprem` data on Tallgrass to create forcing files for SnowModel. I was wondering if the bias corrected P and T have been added to the Hytest Intake Catalogue? 

Also, I would really appreciate a second opinion on my variable selection for the SnowModel Forcing file. SnowModel requires 3-hourly:
- air temperature
- precipitation
- relative humidity
- windspeed
- wind direction
- and elevation, but that is static

To get these variables from C404 I grabbed:
- U10 - for windspeed and direction
- V10 - for windspeed and direction
- PREC_ACC_NC - for precipitation
- Q2 - for relative humidity calculation
- T2 - for air temperature
- PSFC - for relative humidity calculation
- HGT - for elevation of the WRF grid cell.

Do these variables look correct? Lulin mentioned that there is a 3rd wind component, but I could not find that in the CONUS404 variable dictionary.

**Comment by rsignell-usgs** [comment]
Thanks so much @theobarnhart-USGS for raising the issue here!
@pnorton-usgs  what say ye?

Also Theo, @rviger-usgs will be reaching out to you to get any "user story" successes.

**Comment by theobarnhart-USGS** [comment]
Thanks, @rsignell-usgs! The CONUS404 notebooks have been very helpful in facilitating access to the data.

**Comment by rviger-usgs** [comment]
@theobarnhart-USGS we'll connect you with one of the domain experts to confirm your variable choices offline and summarize back here for the record. (and I second Rich's thanks for posting here!).

**Comment by alaws-USGS** [comment]
@rviger-usgs, we were wondering if there were any updates on the data publishing? Thanks!

**Comment by theobarnhart-USGS** [comment]
I'm a little tardy reporting on my discussions with @rviger-usgs and NCAR scientists regarding variable choices. From what I recall, the bias corrected temperature and precipitation can be made internally but its unclear if /when they might be published. Those data I think are also on a different grid from the CONUS404 data so some additional regridding would be needed. I did not end up using the bias corrected data for my use case but I did learn from NCAR scientists that my variable choices were appropriate, and that correction of wind direction was needed. Here is what I used to perform that correction: https://www-k12.atmos.washington.edu/~ovens/wrfwinds.html
